btree/btree.c

```diff
@@ -2356,6 +2359,7 @@ static bool ff_verify(const struct nd *node);
 static void ff_opaque_set(const struct segaddr *addr, void *opaque);
 static void *ff_opaque_get(const struct segaddr *addr);
 static void ff_capture(struct slot *slot, struct m0_be_tx *tx);
+static int  ff_header_size(void);
```

Comment:
same comment as above. Change the name to something which identifies the operation for which this function returns the credit count.

Reply:
Updated.